chainer/optimizers/rmsprop_graves.py

```diff
@@ -0,0 +1,55 @@
+import numpy
+
+from chainer import cuda
+from chainer import Optimizer
```

Comment:
Coding style admits does not admit direct import of Classes (only modules are permitted). So please import `chainer.optimizer` instead of `Optimizer`.